tutorial/creating_packages.rst

```diff
@@ -33,6 +33,12 @@ peculiarities of different types of Conan packages like, for example, header-onl
 libraries, packages for pre-built binaries, packaging tools for building other packages or
 packaging your own applications.
 
+.. note::
```

Comment:
If we don't want to deviate too much the attention to the user actually reading the written tutorial, maybe put sometimes this note below the TOC?